contracts/subdaos/pre-propose/cwd-security-subdao-pre-propose/Cargo.toml

```diff
@@ -23,7 +23,7 @@ cw-utils = { version = "1.0.1" }
 serde = { version = "1.0.175", default-features = false, features = ["derive"] }
 cwd-pre-propose-base = { version = "*", path = "../../../../packages/cwd-pre-propose-base" }
 cwd-voting = { path = "../../../../packages/cwd-voting" }
-neutron-sdk = { package = "neutron-sdk", version = "0.6.1" }
```

Comment:
1) forgot to bump version of the contract itself
2) contract name and dir name are different, which is confusing

Reply:
fixed both